src/components/Modals/ManageAccounts/ManageAccountsModal.tsx

```diff
@@ -157,14 +157,21 @@ export const ManageAccountsModal = () => {
                 onClick={handleClickAddChain}
                 width='full'
                 size='lg'
+                isLoading={isDrawerOpen}
                 isDisabled={disableAddChain}
                 _disabled={disabledProp}
               >
                 {walletConnectedChainIdsSorted.length === 0
                   ? translate('accountManagement.manageAccounts.addChain')
                   : translate('accountManagement.manageAccounts.addAnotherChain')}
               </Button>
-              <Button size='lg' colorScheme='gray' onClick={close} width='full'>
+              <Button
+                size='lg'
+                colorScheme='gray'
+                onClick={close}
+                isDisabled={isDrawerOpen}
```

Comment:
Ditto - disabled makes more sense than loading already, but still unsure about it, z-index should also be all we need UX-wise already, plus clicking away does close the drawer